Fix typo

derekriemer opened the pull request:
There was a typo in browseMode.py

Commit:
Fix typo

derekriemer commented:
on line 92, we have 
isAfterSelection=False #: Is this item positioned after the caret in the document? Used by the elements list to plae its own selection.
what does plae mean? Is this also a typo?

jcsteh commented:
Yes, it's a typo. "plae" should be "place".

Commit:
plae -> place

Commit:
seart -> start

derekriemer commented:
Mind if I put a docstring on the method 313 with signature 
`def addQuickNav(cls, itemType, key, nextDoc, nextError, prevDoc, prevError, readUnit=None):`

derekriemer commented:
Well, actually, that's not a public method.

jcsteh commented:
Go ahead and document it if you wish. It's a class method, but it's 
still meant to be used by subclasses, so it should be documented.

Commit:
add def addQuickNav(cls, itemType, key, nextDoc, nextError, prevDoc, prevError, readUnit=None): script
@jcsteh, I don't understand the readUnit parameter well, (In terms of purpose). If there's a better thing to put, please let me know.

jcsteh commented on source/browseMode.py:
```diff
@@ -311,6 +311,15 @@ def _quickNavScript(self,gesture, itemType, direction, errorMessage, readUnit):
 	def addQuickNav(cls, itemType, key, nextDoc, nextError, prevDoc, prevError, readUnit=None):
+		"""Adds a script for the given quick nav item.
+		@param itemType: The type of item, I.E. "heading" "Link" ...
+		@param key: The quick navigation key to bind to the script. Shift is automatically added for the previous item gesture.
```
Maybe add an example; e.g. h for heading.

jcsteh commented on source/browseMode.py:
```diff
@@ -311,6 +311,15 @@ def _quickNavScript(self,gesture, itemType, direction, errorMessage, readUnit):
+		"""Adds a script for the given quick nav item.
+		@param itemType: The type of item, I.E. "heading" "Link" ...
+		@param key: The quick navigation key to bind to the script. Shift is automatically added for the previous item gesture.
+		@param nextDoc: The input help mode string to bind to the script that yields the next quick nav item.
```
Change "input help mode string" to "command description" or, if you don't think people will get this, "input help mode/input gestures dialog description".

jcsteh commented on source/browseMode.py:
```diff
@@ -311,6 +311,15 @@ def _quickNavScript(self,gesture, itemType, direction, errorMessage, readUnit):
+		@param nextError: The error message if there are no more quick nav items of type itemType in this direction.
+		@param prevDoc: The input help mode string to bind to the script that yields the previous quick nav item.
+		@param prevError: The error message if there are no more quick nav items of type itemType in this direction.
+		@param readUnit: The unit (One of the textInfos.UNIT_* constants to read. Defaults to None
```
Something like this would be more descriptive:

> The unit (one of the textInfos.UNIT_* constants) to announce when moving to this type of item. For example, only the line is read when moving to tables to avoid reading a potentially massive table. If None, the entire item will be announced.

You should probably split this over multiple lines. :)

Commit:
Fix doc strings

jcsteh reviewed (approved):
All good. Thanks!
